traffic-policy/actions/forward-internal/examples/basic-example.mdx

```diff
@@ -0,0 +1,48 @@
+import ConfigExample from "/src/components/ConfigExample.tsx";
+
+### Basic Example
+
+The following [Traffic Policy](/docs/http/traffic-policy/) configuration is an example configuration of the `forward-internal` action.
```

Comment:
here, before we even get to the configuration, tell me what we're setting up. "this example sets up a public endpoint that always forwards traffic to a private endpoint."

may also want to demonstrate that if you only forward some traffic, the non-forwarded traffic goes to the public endpoint's agent tunnel.